playbooks/vars/main.yml

```diff
@@ -31,4 +31,17 @@ octavia_client_cert: "{{ cert_dir }}/client.pem"
 
 neutron_octavia_request_poll_timeout: "{{ '1000' if lookup('env', 'DEPLOY_AIO') == 'yes' else '100' }}"
 octavia_git_install_commit: d9e24e83bbf7d12f8bec16072b28c6ca655690ac #Head of Octavia stable/pike as of 4.1.2018
-requirements_git_install_branch: 99d99fe2e22f4a464415eb0064313b6ebd36906f #HEAD of "stable/pike" as of 4.10.2017
\ No newline at end of file
+requirements_git_install_branch: 99d99fe2e22f4a464415eb0064313b6ebd36906f #HEAD of "stable/pike" as of 4.10.2017
+
+octavia_num_cores: 1000
```

Comment:
Shouldn't this just equal the number of instances?
This is also a minimum and up to the operator and what they pick in the nova flavor, are there not variables for that already? It seems like this should be octavia_num_instances * octavia_flavor_vcpus

Reply:
So, this is the RPC specific part of setting quotas. Do we know that we are running 1 CPU, etc. -- it might make sense to perform this as part of the Octavia OSA install but I didn't go this route to avoid complicating my fork even more...